typescript/src/world-company-remuneration/basket-informations.ts

```diff
@@ -0,0 +1,46 @@
+const K = "K";
+
+/**
+ * Created by thomas on 02/12/2019.
+ * This class is a basket
+ */
+
+export class BasketInformations {
+    // The product of the basket
+    static map: Map<string, number> = new Map<string, number>()
+
+    addProductToBasket(product: string, price: number) {
+        BasketInformations.map.set(product, price)
+    }
+
+    getBasketPrice(inCents: boolean): Number {
+        var v = 0;
+        for (let s of Array.from(BasketInformations.map.values())) {
```

Comment:
Reduce would suffice - instead of adding it up like this

Reply:
Hello, as the repo said, this is only a support for interviewing people, we already know that they are a lot of issues in this code. Thank you for your participation anyway